src/pages/LandingPage/index.tsx

```diff
@@ -82,15 +82,29 @@ const LandingPage = () => {
 
   useEffect(() => {
     // Handle direct route to dataset-playground
-    if (location.pathname === '/dataset-playground') {
+    if (pathname === '/dataset-playground') {
       const element = document.getElementById('dataset-playground');
       if (element) {
         setTimeout(() => {
           element.scrollIntoView({ behavior: 'smooth', });
         }, 100);
       }
     }
```

Comment:
I suppose we cannot delete this because we already shared links with this link?

Reply:
yes, exactly. would have been better to use this approach from the beginning.